modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/edit_content_dashboard_configuration.jsp

```diff
@@ -59,7 +59,7 @@ ContentDashboardAdminConfigurationDisplayContext contentDashboardAdminConfigurat
 						</span>
 
 						<react:component
-							module="js/VocabulariesSelectionBox"
+							module="js/components/VocabulariesSelectionBox"
```

Comment:
I wonder if `VocabulariesSelectionBox.js` should be inside `AuditGraphApp` as it is coupled and serves the chart 🤔

Reply:
Mmmm, it is true that it is related to the graph, but it is also related to the content table. I would leave it under the components folder.